### DISCUSSION TITLE
Personalizar CHECKOUT - Client VAT Number

### DISCUSSION THREAD

**Discussion by AySInformatica** [woocommerce/woocommerce#52104] [Checkout Flow]
How can I customize and include the billing_DNI field to include the DNI/NIF/VAT so that it is later reflected in the invoices?

**Comment by alexflorisca** [comment]
You can use the [additional fields API](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/)

**Comment by nielslange** [comment]
Hello @AySInformatica 👋

As @alexflorisca mentioned, you can use the [additional fields API](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/) for adding fields to the Checkout form. Here's an example code snippet, that should help you getting started:

```php
// Add DNI/NIF/VAT Number below the Order information section on the Checkout block.
add_action(
    'woocommerce_init',
    function () {
        woocommerce_register_additional_checkout_field(
            array(
                'id'       => 'namespace/vat-number',
                'label'    => 'DNI/NIF/VAT Number',
                'location' => 'order',
            )
        );
    }
);
```

For sanitisation, validation, etc., please consult the docs that @alexflorisca mentioned before. You can add this snippet to your site using the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin.
